README.md

```diff
@@ -4,34 +4,57 @@ This library implements the most commonly used Conduce API endpoints.  It can be
 
 `api.py` is the majority of the implementation.  `conduce.py` is a command line utility for invoking the API.  It also servers as a representative implementation of the library.
 
+# Prerequisites
+
+Conduce Python API requires that you have installed the following:
+
+- Python 2.7: https://www.python.org/downloads/
+- pip: https://pip.pypa.io/en/stable/installing/
+
 # Installation
 
 To install using pip:
-
 ```
 pip install git+git://github.com/ConduceInc/conduce-python-api
 ```
-
 or clone the repository and run:
+```
+pip install .
+```
+from the installed directory.
 
+# Using the API
+
+Once you have installed the API, you may use it in a python script by importing the `conduce` module:
+```
+import conduce
+```
+Then you may use API functions, like `create_dataset` as follows:
 ```
-python setup.py install
+conduce.api.create_dataset('my-dataset-name', host='app.condcuce.com', user='email@example.com')
```

Comment:
Stupid question... if the api lives inside an api package, what, if anything, lives directly in the main conduce package?  It seems like an unnecessary indirection when looking through the lens of "this is the conduce api library".

Reply:
Are you talking about `__init__.py`?

`conduce.py` is the command line interface.

Reply:
Just wondering why it's `conduce.api.create_dataset` and not `conduce.create_dataset`